base/dataset.go

```diff
@@ -260,45 +260,6 @@ func RawDatasetRefs(ctx context.Context, r repo.Repo) (string, error) {
 	return builder.String(), nil
 }
 
-// FetchDataset grabs a dataset from a remote source
-func FetchDataset(ctx context.Context, r repo.Repo, ref *reporef.DatasetRef, pin, load bool) (err error) {
```

Comment:
What replaces FetchDataset, didn't seem to find the the alternative?

Reply:
Yeah there _isn't_ an alternative right now. FetchDataset doesn't currently fetch logbook data, and this logic really needs to be moved into the `p2p` package. I'd like to follow up in the very near future with a PR that replaces `FetchDataset`. Will open an issue